src/components/Footer.astro

```diff
@@ -20,19 +20,21 @@
   footer a:hover {
     text-decoration: underline;
   }
+  footer a[target='_blank']::after {
+    display: none;
+  }
 </style>
 
 <footer>
   <h2>Built with ❤️ in Seattle</h2>
   <p>
     This website is built using
-    <a target="_blank" href="https://preactjs.com"> Preact </a>{' '}
-    and{' '}
-    <a target="_blank" href="https://vite.dev/"> Vite </a>{' '}
-    and the code is open sourced on on
+    <a target="_blank" href="https://preactjs.com">Preact</a>,{' '}
+    <a target="_blank" href="https://astro.build">Astro</a>, and{' '}
+    <a target="_blank" href="https://vite.dev/">Vite</a>.
+    {' '}The code is open sourced on{' '}
```

Comment:
Thanks! I was originally doing a rebuild with vite + preact but then changed to astro and forget to update this. Thanks for doing it!